### ISSUE TITLE
Consider using temperature of 0 for greater determinism?

### ISSUE THREAD

**Issue by rattrayalex** [lintrule/lintrule#6]
I have a single new commit, adding the file `bugs.ts` with contents `throw new Error("I am a bug!")`, and the default `rules/no-bugs.md` with contents `don't approve obvious bugs.`

Running `rules check` twice, I get different responses:

```
➜  rules check

$ git diff HEAD^

Found 2 changed files...

   ✔️ PASS  rules/no-bugs.md => rules/no-bugs.md (4122ms)
   FAIL  bugs.ts rules/no-bugs.md
The code contains an obvious bug: a thrown error with the message 'I am a bug!' (9684ms)
  1 rules failed. 
```
and then
```
➜  rules check

$ git diff HEAD^

Found 2 changed files...

   FAIL  bugs.ts rules/no-bugs.md
Obvious bug found: Error 'I am a bug!' is deliberately thrown. (6723ms)
   FAIL  rules/no-bugs.md rules/no-bugs.md
The code snippet is not provided. Please provide a code snippet to evaluate. (7186ms)
  2 rules failed. 
```

I notice that the error message for the obvious bug is different between the two, which is the core of this report.

However, note that there appears to be a separate bug causing the "Please provide a code snippet to evaluate" error message. Let me know if you'd like me to file that separately.

**Comment by Flaque** [comment]
Done!
